contributing.md

```diff
@@ -1,12 +1,10 @@
 # Contributing
```

Comment:
It is **break lines** not **blank lines**. Look for example, the line 2 to 3, or 8 to 9 etc.

Reply:
The founded break lines  were removed.